### DISCUSSION TITLE
Binary code in Github

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67343] [Code Security] [answered]
### Select Topic Area

Question

### Body

Why can't upload the binary code to Github?

**Comment by mehulrajdev** [comment, marked as answer]
To work with binary files on GitHub, it's better to use Git Large File Storage (Git LFS), a Git extension specifically designed for managing large files. Git LFS replaces large binary files with small text pointers in your Git repository, and the actual binary files are stored on a separate server optimized for large files.
